x-pack/plugins/apm/server/routes/services.ts

```diff
@@ -91,10 +95,77 @@ export const serviceAnnotationsRoute = createRoute(() => ({
     const { serviceName } = context.params.path;
     const { environment } = context.params.query;
 
+    let annotationsClient: ScopedAnnotationsClient | undefined;
+
+    if (context.plugins.observability) {
+      annotationsClient = await context.plugins.observability.getScopedAnnotationsClient(
+        request
+      );
+    }
+
     return getServiceAnnotations({
       setup,
       serviceName,
-      environment
+      environment,
+      annotationsClient
+    });
+  }
+}));
+
+export const serviceAnnotationsCreateRoute = createRoute(() => ({
+  path: '/api/apm/services/{serviceName}/annotation',
+  method: 'POST',
+  options: {
+    tags: ['access:apm', 'access:apm_write']
```

Comment:
Users can easily work around this by writing to the index directly, but I wanted to keep things consistent with the agent config APIs. Thoughts?

Reply:
Thoughts around the security model? I think this is good. Now at least it's possible to lock down the index and only give api access.

Reply:
... oh, you can pass in `tags`.. yeah not sure I understand that part.

Reply:
you mean `tags` on an annotation? or `tags` on the route?

Reply:
Disregard the above comments. `tags` for annotations make sense 👍